argument.go

```diff
@@ -188,6 +202,8 @@ func (o *arg) usage() string {
 	switch o.result.(type) {
 	case *bool:
 		break
+	case *int:
+		result = result + " <value>"
```

Comment:
Technically `<value>` is okay, but I would imagine may be not clear enough for whomever reads help message. So maybe `<int>` would be good? Not necessary to change, just a thought.

Reply:
Makes sense to me - thanks!  I'll take a look at the Python `argparse` module and use it as a reference.  If it uses `<int>` or something similar, I'll go with that change for consistency.

Reply:
I am not sure that Python module does even have such message differences. This module inspired by it, but does not really copy it. With help messages I went after what's common on _classic_ CLI tools, which of course may differ one to another. I guess just need to use common sense here. If you (as a dev) feel it will be clear enough value should be numeric, then I am good with that.

Reply:
Turns out it does not, but the error message that gets presented when a non-integer value is passed to an integer parameter is more clear and references `int`.  I'll modify this code to hopefully make it a little more clear and readable.  Thanks again!